connexion/problem.py

```diff
@@ -40,3 +40,10 @@ def problem(status, title, detail, type=None, instance=None, headers=None, ext=N
     return ConnexionResponse(status, mimetype, content_type,
                              body=problem_response,
                              headers=headers)
+
+
+def problem_from_object(exception):
```

Comment:
I'm not sure about the naming here (though it's no worse than `to_problem` which is also odd).

What is the benefit of moving this factory method from `ProblemException.to_problem` into a separate factory function?

Reply:
IMO, I have always viewed exceptions as simply exceptions. I think tracing a response through an exception(and back up to other code) can be very confusing at times. I have always found out easier to think of it as "Okay we now have this exception, let me use this other function to convert it to a response for me". 

Also, any suggestions for the name? I have no opinion on the names, I just picked something that looked similar. If you have a more preferred naming convention, I can change it.